index.js

```diff
@@ -25,19 +25,23 @@ module.exports = {
     else {
 
 
-      const data = await fetch(instagramGraphUrl)
+      const data = await fetch(instagramGraphUrl, {
+        headers: {
+          cookie: `sessionid=${inputs.sessionID}`
+        }
+      })
         .then(res => {
           // ensure that we are only acting on JSON responses
           if(res.headers.get('content-type').includes('application/json')){
             return res.json();
           } else {
-            return null;
+            return res.text();
```

Comment:
This change isn't strictly related to the PR; however, while debugging I found it easier to have the promise return the data no matter what type it is so that I could `console.log` and see what was failing.